app/assets/stylesheets/campaigns.scss

```diff
@@ -54,3 +54,13 @@
   margin: 10px 0 0;
   padding: 5px 30px;
 }
+
+.style-dropdown {
+  margin: 15px 0;
+
+  a, span {
+    &:active {
+      background-color: #17a2b8;
```

Comment:
color must be variable. we have file variables.scss

Reply:
even if we use it one time?